scss/atoms/buttons/_buttons.scss

```diff
@@ -39,49 +39,49 @@
 }
 
 .btn {
-	@include btn(	map-get($ui-btn, margin));
```

Comment:
I thought we were gonna add ui as a prefix in jsass, we are abandoning it completely then ?